developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
+and displaying any information, regardless of where the information comes from. 
+The Info Framework can process data from the database, created 
+through process in memory, or retrieved from an external source.
+
+The Info Framework is a new feature @product-ver@ and currently has limited 
+functionality. The current version is the first set for obtaining and 
+displaying information from external systems or custom data models. The Info 
+framework is also very lightweight, and  does not require you to implement any  
+specific interface to use it. That means you can use it for an object even if 
+you don't have access to the source. 
+
+Rather than create a monolithic framework which requires all of the features to 
+be implemented for the sake of using any, it is a collection of loosely coupled 
+micro-frameworks. This lowers the learning curve and minimizes work for 
+developers.
+
+| Note: Liferay veterans may notice some similarities between the [Asset Framework](asset-framework-link) and Info 
+| Frameworks. The Info Framework doesn't yet provide the full array of features 
+| that Assets provide, but it has many similar characteristics and is simpler 
```

Comment:
This is very vague and not really accurate since it leaves the impression that the Info Framework is aimed as a replacement for the Asset Framework that is simpler. That's not really the case. For several features (tags, categories, ...) the Asset Framework will always be the way to go.

Reply:
I'll change this to be closer to your original text to better differentiate the goal of the Info Framework vs. the Asset Framework. Did you have any other comments? Or was everything else clear and accurate?